code/datums/diseases/asthma_attack.dm

```diff
@@ -0,0 +1,262 @@
+/datum/disease/asthma_attack
```

Comment:
I haven't looked at the viro PRs but this won't have a problem if viro gets removed, right?

Reply:
nope no issues